engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -8,6 +8,7 @@ import better.files._
 import cromwell.core.{CallOutput, WorkflowId}
```

Comment:
I'll remove the changes in this file. It just adds the submission capability of a workflow to the Shadow world via command line, forgot to remove it, was just for my debugging purposes.